x-pack/plugins/cloud/public/fullstory.ts

```diff
@@ -15,9 +15,11 @@ export interface FullStoryDeps {
 }
 
 export type FullstoryUserVars = Record<string, any>;
+export type FullstoryVars = Record<string, any>;
 
 export interface FullStoryApi {
   identify(userId: string, userVars?: FullstoryUserVars): void;
+  setVars(pageName: string, vars?: FullstoryVars): void;
```

Comment:
@afharo do you know if this is gonna impact the EBT RFC, given that it seems we're going to have two 'setVars' APIs on the FS shipper?

Reply:
That's a good point! To be fair, I had it in mind (and made a reference to similar APIs from Segment). I made sure to comment about this API for the FullStory use case. Thanks for raising it 🧡